### ISSUE TITLE
FutureWarning when using extent

### ISSUE THREAD

**Issue by lkiesow** [ResidentMario/geoplot#187]
Using the `extent` parameter, for example like this:

```python
geoplot.pointplot(gdf, …, extent=(-180, -90, 180, 90))
```

…yields a FutureWarning:

```
…/geoplot/geoplot.py:625: FutureWarning:     You are passing non-geometry data to the GeoSeries constructor. Currently,
    it falls back to returning a pandas Series. But in the future, we will start
    to raise a TypeError instead.
  extent = gpd.GeoSeries(self.extent) if self.extent is not None else None
```

While it still works right now, it may be something to take a look at before it breaks.

**Comment by ResidentMario** [comment]
Thanks for reporting! Will look into it.

**Comment by ResidentMario** [comment]
Fixed in #188.
